docs/ember/component-signatures.md

```diff
@@ -21,10 +33,11 @@ export interface SuperTableSignature<T> {
   Args: {
     items: Array<T>;
   };
-  // We accept two named blocks: an optional `header`, and a required
-  // `row`, which will be invoked with each item and its index.
-  Yields: {
-    header?: [];
+  // We accept two named blocks: a parameter-less `header` block
+  // and a `row` block which will be invoked with each item and
+  // its index sequentially.
+  Blocks: {
+    header: [];
```

Comment:
This was a bit of intentional cleanup from a much earlier version of Glint's internals. Today Glint doesn't have a notion of required-ness for blocks, since `{{yield}}`-ing to a block that your consumer didn't actually pass is just a noop. https://github.com/typed-ember/glint/pull/149 has a bit more detail on that